app/konnect/getting-started/configure-mesh.md

```diff
@@ -0,0 +1,106 @@
+
+1. In {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager), click the `example-cp` control plane you just created, and then click **Zones** in the sidebar.
+1. Click **Create Zone**. 
+1. Enter "eu" in the **Name** field for the new zone, and then click **Create Zone & generate token**. 
+    {:.note}
+    > **Note:** The zone name must consist of lower case alphanumeric characters, `.`, or `-`. It must also start and end with an alphanumeric character.
+1. Ensure both [**Zone Ingress**](/mesh/latest/production/cp-deployment/zone-ingress/) and [**Zone Egress**](/mesh/latest/production/cp-deployment/zoneegress/) are enabled.
+1. Follow the instructions that display to set up Helm and a secret token. 
+    Once {{site.konnect_short_name}} finds the newly created zone, it will display it. 
+1. Click **Next**.
+
+You now have a very basic {{site.mesh_product_name}} service mesh added to {{site.konnect_short_name}}. This service mesh can't do anything at the moment until we add services and additional configurations to it.
+
+## Configure `kumactl` to connect to your global control plane
+
+`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can create, read, update, and delete resources in {{site.mesh_product_name}} in Universal/{{site.konnect_short_name}} mode.
+<!--* (not sure if I removed the right line, so keeping in for easy swap 
+Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. -->
+
+You connect `kumactl` to the global control plane in {{site.konnect_short_name}} so that you can run commands against the control plane.
+
+1. In {{site.konnect_short_name}}, select your user icon to open the context menu and click **Personal access tokens**, then click **Generate token**.
+1. Enter "control-plane" in the **Name** field and click **Generate**.
+1. Copy the personal access token that displays.
+1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.
+1. Copy your control plane ID.
+1. Run the following to connect your global control plane to {{site.product_mesh_name}}:
+    ```sh
+    kumactl config control-planes add --address https://us.api.konghq.com/v0/mesh/control-planes/MGCP_ID/api --name mesh-konnect --headers 'authorization=Bearer KPAT'
+    ```
+    * **MGCP_ID:** Replace this with the control plane ID.
+    * **KPAT:** Replace this with the personal access token you created for the control plane.
+
+You can now run commands against your global control plane using `kumactl`. You can see the [`kumactl` command reference](/mesh/latest/generated/cmd/kumactl/kumactl/) for more information about the commands you can use.
```

Comment:
Maybe it's better to run the demo app first, then learn how to setup kumactl and then you can even check it works by doing `kumactl get dataplanes` or apply something and go see it in the GUI.

Reply:
I'll add this suggestion to a list of things we'll look into update for GA: https://konghq.atlassian.net/browse/DOCU-3165